<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QvisMainManager</class>
 <widget class="QWidget" name="QvisMainManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>125</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>10</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>125</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Data Manager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <selectedon>:/Src/FluoRender/UI/icons/FluoRenderIcon.gif</selectedon>
   </iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="MainSplitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QvisSourceManager" name="SourceManager" native="true"/>
     <widget class="QvisDatasetManager" name="DatasetManager" native="true"/>
     <widget class="QvisWorkspaceManager" name="WorkspaceManager" native="true"/>
     <widget class="QvisAnimationController" name="AnimationController" native="true"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QvisSourceManager</class>
   <extends>QWidget</extends>
   <header>QvisSourceManager.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QvisDatasetManager</class>
   <extends>QWidget</extends>
   <header>QvisDatasetManager.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QvisWorkspaceManager</class>
   <extends>QWidget</extends>
   <header>QvisWorkspaceManager.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QvisAnimationController</class>
   <extends>QWidget</extends>
   <header>QvisAnimationController.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
